### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by dariomws — APPROVED** [review]
@isrumnon, the same note is there for the -Identity, -Filter, and Output sections. Do we need to update it only in the Notes section?

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -637,7 +637,7 @@ In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the
 
 ## NOTES
 
-These changes are currently only rolled out in commercial environments and are **not** applicable to government environments.
+These changes are currently only rolled out in commercial environments at present and will be applicable to government environments starting 31st of March 2023.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:640]:
```suggestion
These changes are only rolled out in commercial environments at present and will be applicable to government environments starting March 31, 2023.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Comment by isrumnon** [comment]
> @isrumnon, the same note is there for the -Identity, -Filter, and Output sections. Do we need to update it only in the Notes section?

Thanks for pointing that out @dariomws I'll add the same note to those other sections

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ Accept wildcard characters: False
 ### -Identity
 Indicates the Identity of the user account to be retrieved.
 
-For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are currently only rolled out in commercial environments and are currently **not** applicable to government environments):
+For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that These changes are only rolled out in commercial environments at present and will be applicable to government environments starting March 31, 2023.):
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:145]:
```suggestion
For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments at present and will be applicable to government environments starting March 31, 2023.):
```

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]
Only @dariomws's suggested change.

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Gentle reminder. Thanks

**Pull request merged**